src/fam-api/fam.cpp

```diff
@@ -776,20 +822,27 @@ int fam::Impl_::validate_fam_options(Fam_Options *options) {
 
     if (options && options->allocator)
         famOptions.allocator = strdup(options->allocator);
+    else if (!config_file_fam_options.empty() &&
+             config_file_fam_options.count("allocator") > 0)
```

Comment:
Do we have a config file option called allocator, it is now openfam model, Correct ?